README.md

```diff
@@ -11,3 +12,29 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+We want to show that $O(\log_2(n))$ is equivalent to $O(\log_5(n))$ up to a constant factor, which means there exists a constant $c$ such that:
```

Comment:
No, you need to show that they are equivalent, not subject to a constant factor.